module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -207,8 +207,8 @@ namespace module::purpose {
         on<Trigger<EnableIdle>>().then([this] {
             // Stop all tasks and stand still
             emit<Task>(std::unique_ptr<FindPurpose>(nullptr));
+            emit<Task>(std::unique_ptr<FallRecovery>(nullptr));
```

Comment:
No fall recovery on idle? Is idle like really idle?

Reply:
it's a state we have for making the robot stop entirely when we pick it up